frontend/webservice/circuit.cgi

```diff
@@ -310,6 +310,12 @@ sub provision {
             return;
         }
 
+        if(defined $interface->provisionable_bandwidth && !($ep->{bandwidth} + $interface->{utilized_bandwidth} <= $interface->provisionable_bandwidth)){
```

Comment:
For some reason my mind went blank and I said to use `<=`.  What you had first with `>` was better.